variables.tf

```diff
@@ -70,3 +70,8 @@ variable "labels" {
   description = "The resource labels to represent user provided metadata."
   default     = {}
 }
+
+variable "enable_apis" {
+  description = "Enable required APIs for Cloud Memorystore."
+  default     = true
```

Comment:
Boolean variable defaults should be quoted because terraform will coerce `true` to `1`, and this behavior will change on 0.12.